File: aretrofitdemo/doc/retrofit30/spring-boot-example-v1/src/main/java/com/mxdl/springboot/bean/AddUserResponse.java
package com.mxdl.springboot.bean;

public class AddUserResponse extends BaseResponse {
    private User user;

    public AddUserResponse(String msg, int code) {
        super(msg, code);
    }

    public AddUserResponse(String msg, int code, User user) {
        super(msg, code);
        this.user = user;
    }

    @Override
    public String toString() {
        return "AddUserResponse{" +
                "user=" + user +
                '}';
    }
}

File: aretrofitdemo/doc/retrofit30/retrofit-kotlin-example-v1/app/src/main/java/com/mxdl/retrofit/bean/AddUserResponse.kt
package com.mxdl.retrofit.bean

/**
 * Description: <AddUserResponse><br>
 * Author:      mxdl<br>
 * Date:        2020/8/9<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
class AddUserResponse :BaseResponse {
    var user :User

    constructor(msg: String, code: Int, user: User) : super(msg, code) {
        this.user = user
    }
}